modules/centralized-logging/variables.tf

```diff
@@ -69,3 +74,9 @@ variable "kms_key_protection_level" {
   type        = string
   default     = "HSM"
 }
+
+variable "data_access_logs_enabled" {
+  description = "Enable Data Access logs of types DATA_READ, DATA_WRITE for all GCP services. Enabling Data Access logs might result in your organization being charged for the additional logs usage. See https://cloud.google.com/logging/docs/audit#data-access The ADMIN_READ logs are enabled by default."
```

Comment:
"... for all GCP services *in the projects under the provided `parent_folder`*. "

Reply:
Good catch!